internal/pkg/template/templates/workloads/services/rd-web/cf.yml

```diff
@@ -193,4 +199,8 @@ Resources:
 
 {{- if ne .EnvControllerLambda ""}}
 {{include "env-controller" . | indent 2}}
-{{- end}}
\ No newline at end of file
+{{- end}}
+
+{{- if (and (contains .FeatureFlags "VPCConnector") (eq .Network.SubnetsType "PrivateSubnets")) }}
```

Comment:
> `(eq .Network.SubnetsType "PrivateSubnets")` 

Is this meant to be an extra safeguard in addition to manifest validation against people that wants to put "public" for vpc?

Reply:
The `FeatureFlags` expression is indeed meant to be an additional safeguard so that if anybody enters `placement: 'private'` today we don't create these resources 😅 

The `Network` expression is mandatory and will remain there even after we remove the `FeatureFlags` expression. We want to ensure that these resources are created only while connecting to private subnets